File: C#-OOP-Basics/Problems Course Instance 2018/DefiningClasses/OldestFamilyMember/Person.cs
public class Person
{
    private string name;
    private int age;

   public Person()
    {
        this.Age = 1;
        this.Name = "No name";
    }

   public Person(int age) : this()
    {
        this.Age = age;
    }

   public Person(int age, string name)
    {
        this.Age = age;
        this.Name = name;
    }

    public string Name { get => name; set => name = value; }
    public int Age { get => age; set => age = value; }
}

File: Java-OOP-Advanced/Problems Course Instance 2016_2017/src/Lesson03IteratorsAndComparators/Exercises/pr05_comparing_objects/Person.java
package Lesson03IteratorsAndComparators.Exercises.pr05_comparing_objects;

public class Person implements Comparable<Person> {

    private String name;
    private int age;
    private String town;

    public Person(String name, int age, String town) {
        this.name = name;
        this.age = age;
        this.town = town;
    }

    @Override
    public int compareTo(Person other) {
        int cmp = this.name.compareTo(other.name);
        if (cmp == 0) {
            cmp = Integer.compare(this.age, other.age);
        }
        if (cmp == 0) {
            cmp = this.town.compareTo(other.town);
        }
        return cmp;
    }
}
